lib/ldapUser.class.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+/*
+ * This file is part of the Access to Memory (AtoM) software.
+ *
+ * Access to Memory (AtoM) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Access to Memory (AtoM) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Access to Memory (AtoM).  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+include sfConfig::get('sf_root_dir') .'/vendor/password_compat/password.php';
```

Comment:
As that's an SPL autoloader too, I don't think it'll work. It looks like there's an RFC for autoloading functions rather than classes (https://wiki.php.net/rfc/autofunc) but is has been opposed (as of PHP 5.4: http://www.serverphorums.com/read.php?7,597588). One trick people use is making functions into static methods of classes, but password_compat doesn't do that. :(